templates/extensions/the-graph/README.md.args.mjs

```diff
@@ -0,0 +1,247 @@
+export const extraContents = `## 🚀 Setup The Graph Integration
+
+Now that we have spun up our blockchain, started our frontend application and deployed our smart contract, we can start setting up our subgraph and utilize The Graph!
+
+> Before following these steps be sure Docker is running!
+
+&nbsp;
+
+#### ✅ Step 1: Clean up any old data and spin up our docker containers ✅
+
+First run the following to clean up any old data. Do this if you need to reset everything.
+
+\`\`\`
+yarn clean-node
+\`\`\`
+
+> We can now spin up a graph node by running the following command… 🧑‍🚀
+
+\`\`\`
+yarn run-node
+\`\`\`
+
+This will spin up all the containers for The Graph using docker-compose. You will want to keep this window open at all times so that you can see log output from Docker.
+
+> As stated before, be sure to keep this window open so that you can see any log output from Docker. 🔎
+
+> NOTE FOR LINUX USERS: If you are running Linux you will need some additional changes to the project.
+
+##### Linux Only
+
+Update your package.json in packages/hardhat with the following command line option for the hardhat chain.
+
+\`\`\`
+"chain": "hardhat node --network hardhat --no-deploy --hostname 0.0.0.0"
+\`\`\`
+
+Save the file and then restart your chain in its original window.
+
+\`\`\`
+yarn chain
+\`\`\`
+
+You might also need to add a firewall exception for port 8432. As an example for Ubuntu... run the following command.
+
+\`\`\`
+sudo ufw allow 8545/tcp
+\`\`\`
+
+&nbsp;
+
+#### ✅ Step 2: Create and ship our Subgraph ✅
+
+Now we can open up a fourth window to finish setting up The Graph. 😅 In this forth window we will create our local subgraph!
```

Comment:
for me its fifth 😄 . Chain, deploy, start, run-node

Reply:
Ohh lol yes ! Updated at https://github.com/scaffold-eth/create-eth/pull/25/commits/6272c812dd659c95fb677efcf435a8dc9f9cf54c